packages/cloud-paks/README.md

```diff
@@ -4,26 +4,26 @@
 
 ## Getting started
 
-To install `@carbon/ibm-cloud-paks` in your project, you will need to run the
-following command using [npm](https://www.npmjs.com/):
+To install `@carbon/ibm-cloud-paks` in your project, you'll need to run one of
```

Comment:
Concerned that we are creeping closer to accidentally publishing a package called ibm-cloud-paks. I am tempted just to go with 'carbide-components' and see if anyone higher up complains.